src/styles/index.scss

```diff
@@ -0,0 +1,77 @@
+#app {
+    font-family: Montserrat, sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    font-size: 16px;
+    color: #282828;
+  }
+
+@mixin mobile {
+  @media screen and (min-width: 480px) {
+    @content;
+  }
+}
+
+@mixin tablet {
+  @media screen and (min-width: 768px) {
+    @content;
+  }
+}
+
+@mixin desktop {
+  @media screen and (min-width: 1200px) {
+    @content;
+  }
+}
+
+.container {
+  width: 100%;
+  margin: 0 auto;
+  padding: 30px 0px;
```

Comment:
I would recommend reducing the padding value on mobile to 15px or 20px and using 30px 30px from screens (min-width: 480px)